charts/latest/azuredisk-csi-driver/values.yaml

```diff
@@ -25,11 +25,20 @@ image:
     pullPolicy: IfNotPresent
 
 serviceAccount:
-  create: true
+  controller:
+    create: true # When true, a service account will be created for you. Set to false if you want to use your own.
```

Comment:
that's redundant, could just use `serviceAccount.controller.name` instead since it would always be provided either by default value or user.

Reply:
I was using this "create" value in each of the templates for creating the SA (e.g. serviceaccount-csi-azuredisk-controller.yaml), since that was how it was previously structured. Should I instead do a check, if the name != default then create the service account?

Reply:
since there is already `serviceAccount.create` and `rbac.create`, only need to specify `controller.name`, `node.name`, `snapshotcontroller.name`, if user wants to bring their own serviceAccount, they need to `--set serviceAccount.create=false --set rbac.create=false --set serviceAccount.controller.name=x --set serviceAccount.node.name=x --set serviceAccount.snapshotcontroller.name=x`